floodlight/io/catapultApi.py

```diff
@@ -256,28 +271,40 @@ def get_meta_data(response_data_json_list: List[Dict]) -> Tuple[Dict[str, Dict[s
         "number": "jersey",
     }
 
-    for data_i in response_data_json_list:
-        for entry in data_i["data"]:
-            t.append(int(entry["ts"]) * 100 + int(entry["cs"]))
-
+    from collections import Counter

```

Comment:
Please put imports at the top of the file.

Reply:
done